S15/tests/dbbench/run-all.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -x
+set -e
+
+FILES=$(ls queries/1_*.ini)
+OUTDIR="queries/out"
+for f in $FILES 
+do
+    echo "Processing $f file..."
+    outfile=$(basename "$f")
+    outfile=$(echo "$outfile" | sed 's/\.ini$/\.stats/')
+    mkdir -p "$OUTDIR"
+    dbbench --database=LSST --query-stats-file="$OUTDIR/${outfile}" --username=qsmaster --host=ccqserv100 --port=4040 "$f" 
```

Comment:
The code looks fine to me, although I'm unfamiliar with `dbbench`. One thing I'd note is that the test seems to be hard coded for the `LSST` dataset only right now, and it might be a pain to change all the individual `query.ini` files for the correct parameters for tests on other datasets.